modules/help_editor/jsx/help_editor.js

```diff
@@ -0,0 +1,176 @@
+import FilterForm from 'FilterForm';
+import Loader from 'Loader';
+
+/**
+ * Help Editor Archive Page.
+ *
+ * Serves as an entry-point to the module, rendering the whole react
+ * component page on load.
+ *
+ * Renders Help Editor main page consisting of FilterForm and
+ * StaticDataTable components.
+ *
+ * @author LORIS Team
+ * @version 1.0.0
+ *
+ * */
+class HelpEditor extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoaded: false,
+      filter: {},
+      data: {}
+    };
+
+    // TODO: refs should be deprecated in future refactoring.
+    /**
+     * Set filter to the element's ref for filtering
+     */
+    this.filter = null;
+    this.setFilterRef = element => {
+      this.filter = element;
+    };
+
+    /*
+     * Bind component instance to custom methods
+     */
+    this.fetchData = this.fetchData.bind(this);
+    this.updateFilter = this.updateFilter.bind(this);
+    this.resetFilters = this.resetFilters.bind(this);
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   */
+  fetchData() {
+    $.ajax(this.props.DataURL, {
+      method: 'GET',
+      dataType: 'json',
+      success: data => {
+        // FIXME: Remove the following line of code as soon as hiddenHeaders is
+        // accepted as a prop by the StaticDataTable Component.
+        loris.hiddenHeaders = data.hiddenHeaders || [];
```

Comment:
is there any reason to set this here if you are going to read the hidden headers from state (L:98)?

Reply:
Yeah, it's for access by the `StaticDataTable` because it does not yet access `hiddenHeaders` because that was pushed to major. So this is just a place-holder until major and minor get merged if that makes sense.